styleguide/source/_patterns/02-molecules/references.twig

```diff
@@ -5,17 +5,13 @@
     {% for reference in references %}
       <li id="reference_{{ number }}" class="joe__references__item">
```

Comment:
@aCyborg instead of setting this as a variable you can access the index of the loop using: {{ loop.index }}. Using this you can remove number from the template and json.

Reply:
oh awesome, thanks